doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -125,6 +125,11 @@ Applications
 
 This section provides detailed lists of changes by :ref:`application <applications>`.
 
+* Matter bridge application
```

Comment:
Please move this to the Matter bridge section (around Line 170)

Reply:
Oh, I forgot that we have such a section :)